lib/adapter_base.coffee

```diff
@@ -94,10 +95,10 @@ class Adapter
    * @return {Promise}
   ###
   renderFile: (file, opts = {}) ->
-    opts = _.clone(opts, true)
+    opts = clone(opts, true)
     (new File(file))
       .read(encoding: 'utf8')
-      .then _.partialRight(@render, _.extend({ filename: file }, opts)).bind(@)
+      .then partialRight(@render, Object.assign({ filename: file }, opts)).bind(@)
```

Comment:
This breaks compatibility with node `0.12`, just a note that we need to clearly mention this will break in the release notes if you are on an older version. Which I'm ok with, we just need to be clear about it.